packages/components/src/button/README.md

```diff
@@ -134,7 +134,7 @@ Name | Type | Default | Description
 `isDestructive` | `bool` | `false` | Renders a red text-based button style to indicate destructive behavior.
 `isLarge` | `bool` | `false` | Increases the size of the button.
 `isSmall` | `bool` | `false` | Decreases the size of the button.
-`isToggled` | `bool` | `false` | Renders a toggled button style.
+`isPressed` | `bool` | `false` | Renders a pressed button style.
```

Comment:
I didn't keep the old name because it only applied the `is-toggled` class name which was never styled. I can bring it back, but I don't feel like it's a good idea to keep it. It's only confusing.